packages/manager/apps/dedicated/client/app/dedicated/dedicated-server/servers/ftp-backup/access/dedicated-server-ftp-backup-access-add.routing.js

```diff
@@ -0,0 +1,19 @@
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('app.dedicated-server.server.ftpBackup.addAccess', {
+    url: '/addAccess',
+    views: {
+      modal: {
+        component: 'dedicatedServerFtpBackupAccessAdd',
+      },
+    },
+    params: {
+      results: [],
+    },
+    layout: 'modal',
+    resolve: {
+      goBack: /* @ngInject */ (goToFtpBackUpStorage) => goToFtpBackUpStorage,
```

Comment:
Shouldn't reload and banner management be added here in the event of success or error?

Reply:
Seen together no need to take care this.